server/src/models/Review.js

```diff
@@ -0,0 +1,46 @@
+const Model = require("./Model")
+
+class Review extends Model {
+  static get tableName() {
+    return "reviews"
+  }
+
+  static get relationMappings() {
+    const { Trail, User } = require("./index")
+
+    return {
+      trails: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: Trail,
+        join: {
+          from: "reviews.trailId",
+          to: "trails.id",
+        },
+      },
+
+      users: {
```

Comment:
A Review also belongs to one `user`, so the association is singular

Reply:
done